11-carpentries.md

```diff
@@ -3,143 +3,219 @@ layout: page
-was born, as a separate "sister" organization to Software Carpentry.  
-
-Data Carpentry and Software Carpentry share some aspects of their operations, long-term 
-goals, and community structure: 
-* both run two-day workshops taught by volunteer instructors
-* both target the research community, aiming to fill gaps in current training structures
-* both focus on computational skills
-
-Data and Software Carpentry differ in their major content and primary audience: 
-
-* Data Carpentry workshops focus on best practices surrounding data.  The audience 
-for a DC workshop is not someone who wants to learn about coding, but rather someone 
-who has a lot of data and doesn't know what to do with it.  The DC workshops are: 
-	* domain specific
-	* a full two-day curriculum - each "domain" workshop has a two-day curriculum 
-	that centers around a single data set, so separate domain workshops cannot be 
-	mixed and matched.  
-	* aimed at pure novices
+In 2013, members of the Software Carpentry community identified a need
+for training aimed at computational novices that would teach
+researchers how to properly handle their data.  This led to the
+creation of [Data Carpentry][dc].  While separate, the two
+organization share many aspects of their operations, long-term goals,
+and community structure:
+
+*   Both run two-day workshops taught by volunteer instructors.
+*   Both strive to fill gaps in current training for researchers.
+*   Both focus on computational skills.
+
+However, they differ in their content and intended audience.
+Data Carpentry workshops focus on best practices surrounding data.
+Its learners are not people who want to learn about coding,
+but rather those who have a lot of data and don't know what to do with it.
+Accordingly, Data Carpentry workshops:
+
+*   are aimed at pure novices,
+*   domain-specific, and
+*   present a full two-day curriculum centered around a single data set.
 	
-* Software Carpentry workshops focus on best practices surrounding software and 
-code development.  The audience for a SWC workshop is someone who has done a little 
-bit of programming, or has used computational tools primarily inside a GUI.  The 
-SWC workshops are: 
-	* not domain specific
-	* modular - each Software Carpentry lesson is standalone.  Each workshop must cover 
-	at least one lesson each in shell, version control, and a scripting language (python or R)
-	to be considered an official SWC workshop (more details below).  There are various 
-	additional lessons that can be added to these three base lessons.  
-	* aimed at a level slightly above novice, maybe novice+
-
-For more details on the different requirements for running Software and Data Carpentry 
-workshops, or being a fully certified Software/Data Carpentry instructor, see below.  
+Software Carpentry workshops focus on best practices for software development and use.
+Its workshops are:
+
+*   intended for people who have done some programming, but want to do better,
```

Comment:
I think this should be worded as "intended for people who need help to program more effectively to solve their computational challenges"

We know that we still get novices, and those novices have programming needs. We're working on filling that gap for those who have never programmed so I'd just reword to be a tad more open to novices.